src/interfaces/payment-interfaces/fees-interface.ts

```diff
@@ -13,7 +13,7 @@ interface FeesIF {
     total?: number
 }
 
-const emptyFees: FeesIF = {
+export const EmptyFees: FeesIF = {
```

Comment:
PascalCase has been our style for "non variables".

Reply:
👍